src/shared/pipes.tsx

```diff
@@ -59,6 +59,13 @@ export const emDashPipe = (str: string): string => {
   return str.trim() === '' ? '—' : str;
 };
 
+// Formats the user into a full name as a project lead/manager
+export const userPipe = (msg: string, user: User): string => {
+  const userID = ' "' + user.userId.toString() + '"';
+  const newDetail = msg.replace(userID, ': ' + fullNamePipe(user));
+  return newDetail;
```

Comment:
What do you think of this alternative code? It should behave the same, but I wanna let you choose which style you prefer. I'm using template strings with the backtick \` to build a string combined with some TS

```suggestion
  return msg.replace(` "${user.userId}"`, `: ${fullNamePipe(user)}`);
```

Reply:
I see. It looks more simple and concise code and passed all of my tests. Definitely looks good to me, and I would prefer this code style (I was just not familiar with the backtick symbol in TS). However, I will also wait for Reid's comment on this too.